From 8f7898fa6da8e951fff9168be82e511f3b2b52c0 Mon Sep 17 00:00:00 2001
From: Robert Ancell <robert.ancell@canonical.com>
Date: Wed, 17 Apr 2019 10:26:04 +1200
Subject: [PATCH 26/28] odrs: Only show reviews from the same distribution

---
 plugins/odrs/gs-plugin-odrs.c | 6 ++++++
 1 file changed, 6 insertions(+)

Index: gnome-software/plugins/odrs/gs-plugin-odrs.c
===================================================================
--- gnome-software.orig/plugins/odrs/gs-plugin-odrs.c
+++ gnome-software/plugins/odrs/gs-plugin-odrs.c
@@ -298,6 +298,7 @@ gs_plugin_odrs_parse_reviews (GsPlugin *
 			      gssize data_len,
 			      GError **error)
 {
+	GsPluginData *priv = gs_plugin_get_data (plugin);
 	JsonArray *json_reviews;
 	JsonNode *json_root;
 	guint i;
@@ -363,6 +364,11 @@ gs_plugin_odrs_parse_reviews (GsPlugin *
 			return NULL;
 		}
 
+		if (g_strcmp0 (json_object_get_string_member (json_item, "distro"), priv->distro) != 0) {
+			g_debug ("Ignoring review from distro %s", json_object_get_string_member (json_item, "distro"));
+			continue;
+		}
+
 		/* create review */
 		review = gs_plugin_odrs_parse_review_object (plugin,
 							     json_item);
