diff --git a/test cases/frameworks/17 mpi/meson.build b/test cases/frameworks/17 mpi/meson.build
index 75b463cc..7bf4e7ae 100644
--- a/test cases/frameworks/17 mpi/meson.build	
+++ b/test cases/frameworks/17 mpi/meson.build	
@@ -2,6 +2,8 @@ project('mpi', 'c', 'cpp', default_options: ['b_asneeded=false'])
 
 method = get_option('method')
 
+error('MESON_SKIP_TEST openmpi binaries do not work when run in pbuilder for some reason.')
+
 cc = meson.get_compiler('c')
 mpic = dependency('mpi', language : 'c', required : false, method : method)
 if not mpic.found()
